### DISCUSSION TITLE
Create a limited group to view the articles

### DISCUSSION THREAD

**Discussion by cairoapcampos** [requarks/wiki#6588] [Help / Questions]
I created a group called “Suporte” and I want it to have access to the home page and to the articles defined by me in the navigation. The first configuration performed is to give access to the desired pages for the group:

![2023-07-20_17-52](https://github.com/requarks/wiki/assets/21186839/f9ef401d-fe2e-4af5-a6f0-68279b2b04ef)

When logging in with a user from the group, I was able to view the pages in the side menu:

![2023-07-20_17-45](https://github.com/requarks/wiki/assets/21186839/f17f8ed2-5c39-4f12-942a-3f18ecc4b59b)

However, in the search field it is possible to find other pages that the user should not have access to. User can read found pages. I wanted the search user to find only pages that I allowed.

![2023-07-20_17-46](https://github.com/requarks/wiki/assets/21186839/8dd09e83-824a-47dd-8e14-dac71f7e17e5)

Below is an image of rules added to the "Suporte" group.

![2023-07-20_17-48](https://github.com/requarks/wiki/assets/21186839/418b7e1d-b2da-458f-ae26-dce34eac9980)

In addition to only reading the pages I've allowed, I'd like users in this group to create their own articles in their own directory. 
Would it be possible to make the changes I want?

**Comment by P4mes4n** [comment]
Don't try and use the navigation panel to limit what people can access.
Use the permissions and page rules, as this is exactly what they are for.
With your current permission setup, this group can read all articles in your wiki, but can't create any.

Possible Setup, using a `guides` and `content` directory:
Allow: read pages, assets; Path starts with: `/guides`
Allow: read and write pages, assets; Path starts with: `/content`

This way you can create guides that people can read in a dedicated path and users of that group can create their own content in another path.
